realm/realm-library/src/main/java/io/realm/internal/SharedRealm.java

```diff
@@ -175,6 +192,16 @@ public boolean isInTransaction() {
         return nativeIsInTransaction(nativePtr);
     }
 
+    public void setSchemaVersion(long schemaVersion) {
+        // FIXME migrate to ObjectStore
```

Comment:
I found `set_schema_version()` in `object_store.cpp`. It does almost same thing here.